bin/utils/update_browsers.js

```diff
@@ -0,0 +1,75 @@
+/* global __dirname */
+
+const fs = require('fs');
+const path = require('path');
+const browserslist = require('browserslist');
+const semver = require('semver');
+const diff = require('diff');
+const chalk = require('chalk');
+
+(() => {
+  'use strict';
+  const README_PATH = path.join(__dirname, '..', '..', 'README.md');
+
+  const productionQuery = browserslist
+    .loadConfig({
+      env: 'production',
+      path: path.resolve('.'),
+    })
+    .join(', ');
+  const productionURL = `https://browserl.ist/?q=${encodeURIComponent(productionQuery)}`;
+
+  const template = browsers => `
+  | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/firefox/firefox_48x48.png" alt="Firefox" width="24px" height="24px" /></br>Firefox | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/chrome/chrome_48x48.png" alt="Chrome" width="24px" height="24px" /></br>Chrome | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/safari/safari_48x48.png" alt="Safari" width="24px" height="24px" /></br>Safari | <img src="https://raw.githubusercontent.com/alrra/browser-logos/master/src/edge/edge_48x48.png" alt="Edge" width="24px" height="24px" /></br>Edge |
```

Comment:
`</br>`? 🙈

Reply:
BUT HEY, IT WORKS!
Not sure how that got there. Must have been my editor trying to close the tag.